README.md

```diff
@@ -10,25 +10,166 @@
 
-This is a command line tool (CLI) that help you backup your published Content Model, Content and Assets or move them to a new Contentful space. _It will support Editor Interfaces, Webhooks and Roles & Permissions in a future version._
+This is a library that help you backup your Content Model, Content and Assets or move them to a new Contentful space. _It will support Roles & Permissions in a future version._
 
 To import your exported data, please refer to the [contentful-import](https://github.com/contentful/contentful-import) repository.
 
-## Installation
+## :exclamation: Usage as CLI
+We moved the CLI version of this tool into our [Contentful CLI](https://github.com/contentful/contentful-cli). This allows our users to use and install only one single CLI tool to get the full Contentful experience.
 
-We recommend the installation of this CLI via npm:
+Please have a look at the [Contentful CLI export command documentation](https://github.com/contentful/contentful-cli/tree/master/docs/space/export) to learn more about how to use this as command line tool.
+
+
+## :cloud: Installation
 
 ```bash
-npm install -g contentful-export
+npm install contentful-export
+```
+
+## :hand: Usage
+
+
+While this tool is intended for use as a command line tool, you can also use it as a Node library:
+
+```javascript
+const contentfulExport = require('contentful-export')
+
+const options = {
+  spaceId: '<space_id>',
+  managementToken: '<content_management_api_key>',
+  ...
+}
+
+contentfulExport(options)
+  .then((result) => {
+    console.log('Your space data:', result)
+  })
+  .catch((err) => {
+    console.log('Oh no! Some errors occurred!', err)
+  })
+```
+
+
+
+### Querying
+
+To scope your export, you are able to pass query parameters. All search parameters of our API are supported as documented in our [API documentation](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters).
+
+```javascript
+const options = {
+  spaceId: '<space_id>',
+  managementToken: '<content_management_api_key>',
+  queryEntries: 'content_type=<content_type_id>'
+}
+
+contentfulExport(options)
+...
+```
+
+The Export tool also support multiple inline queries.
+
+```javascript
+const options = {
+  spaceId: '<space_id>',
+  managementToken: '<content_management_api_key>',
+  queryEntries: [
+    'content_type=<content_type_id>',
+    'sys.id=<entry_id>'
+  ]
+}
+
+contentfulExport(options)
+...
+```
+
+`queryAssets` uses the same syntax as `queryEntries`
+
+### Export an environment
+
+```javascript
+const options = {
+  spaceId: '<space_id>',
+  managementToken: '<content_management_api_key>',
+  environment: '<environment-id>'
```

Comment:
nitpicky, but let's make sure it's an underscore: `<environment_id>`

also, shouldn't the argument name be `environmentId:` instead of `environment:`

Reply:
in fact, i can see in the configuration options below that it _is_ supposed to be `environmentId` 😉